src/widget/wlibrarysidebar.cpp

```diff
@@ -196,7 +195,7 @@ void WLibrarySidebar::keyPressEvent(QKeyEvent* event) {
 }
 
 void WLibrarySidebar::selectIndex(const QModelIndex& index) {
-    QItemSelectionModel* pModel = new QItemSelectionModel(model());
+    auto  pModel = new QItemSelectionModel(model());
```

Comment:
No, both is allowed. I think I have just found a reason for the double space. 
clang-tidy replaces `"QItemSelectionModel *"` by `"auto "` regardless if there is already a space between the * and the variable name or not.